libsplinter/src/service/network/handlers.rs

```diff
@@ -68,3 +68,134 @@ impl Handler for ServiceMessageHandler {
             })
     }
 }
+
+/// A service registry stores the service information provide by service components.  This can be
+/// used to add or remove service registration information.
+pub trait ServiceRegistry {
```

Comment:
Since we have a registry as a top-level component (was node registry, now Splinter registry), this probably needs a new name to avoid confusion.

Reply:
Renamed to ServiceInstances